packages/survey/src/Question/Item/models/ItemProps.ts

```diff
@@ -0,0 +1,21 @@
+export type OptionObj = {
+  /* the value */
+  value: string
+  /* label we should display (if selected) */
+  label: string
+  /* image to show */
+  imageSrc?: string
+}
+
+export type ItemProps = {
+  /* which type we should use */
+  itemType: 'checkbox' | 'radio'
+  /* the color used when checked is true */
+  activeColor: string
+  /* chakra controlled value */
+  isChecked?: boolean
+  /* show or hide labels */
+  withLabels: boolean
+  /* show or hide images */
+  withImages: boolean
+} & Pick<OptionObj, 'value' | 'label' | 'imageSrc'>
```

Comment:
In all other components, we have used whole sentences as description.

E.g.

```ts
export type SettingsMenuButtonProps = {
  /** The list of menu items. */
  menuItems: MenuListItem[]
  /** The color of the `IoEllipsisVertical` icon. */
  color?: ColorProps['color']
  /** The size of the `IoEllipsisVertical` icon. */
  fontSize?: 'sm' | 'md' | 'lg'
}
```

In my opinion, it would be nice to stay consistent.

This is also true for other parts of the PR.